### ISSUE TITLE
configuration error when "repository" is used

### ISSUE THREAD

**Issue by dylang** [renovateapp/renovate#1395]
When using `repositories` there is a configuration option `repository` - however it is not documented in https://github.com/renovateapp/renovate/blob/master/lib/config/definitions.js, so an error is logged.

```
DEBUG: File config
       "config": {
         "endpoint": "https://github.enterprise/api/v3/",
         "token": "***********",
         "repositories": [
           {
             "repository": "org1/repo1"
           },
           {
             "repository": "org2/repo2"
           }
         ]
       }
DEBUG: CLI config
       "config": {"logLevel": "debug"}
DEBUG: Env config
       "config": {}
ERROR: Found config errors
       "errors": [
         {
           "depName": "Configuration Error",
           "message": "Invalid configuration option: `repository`"
         },
         {
           "depName": "Configuration Error",
           "message": "Invalid configuration option: `repository`"
         }
       ]
````

**Comment by rarkins** [comment]
You can use a simple string array if that's all you need, e.g. like:
         "repositories": ["org1/repo1", "org2/repo2"]

However "repository" should be added to the allowed/ignored list so you don't get this error.

**Comment by rarkins** [comment]
@dylang should be fixed in renovate@10.51.1
